src/elements/DetailsMenu/DetailsMenu.js

```diff
@@ -0,0 +1,210 @@
+import React, { useCallback, useEffect, useState, useRef } from 'react'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+import { FUNCTIONS_PAGE } from '../../constants'
+
+import { ReactComponent as Arrow } from '../../images/arrow.svg'
+
+import './detailsMenu.scss'
+
+const DetailsMenu = ({
+  detailsMenu,
+  hash,
+  id,
+  iter,
+  match,
+  name,
+  onClick,
+  page
+}) => {
+  const [arrowsAreHidden, setArrowsAreHidden] = useState(true)
+  const [scrolledWidth, setScrolledWidth] = useState(0)
+  const [rightArrowDisabled, setRightArrowDisabled] = useState(false)
+  const tabsWrapperRef = useRef()
+  const tabsRef = useRef()
+  const menuOffsetHalfWidth = 2
+  const tabOffset = 1.5
+
+  const leftArrowClassNames = classnames(
+    'details-menu__arrow',
+    'details-menu__arrow_left',
+    arrowsAreHidden && 'details-menu__arrow_hidden',
+    scrolledWidth === 0 && 'details-menu__arrow_disabled'
+  )
+  const rightArrowClassNames = classnames(
+    'details-menu__arrow',
+    'details-menu__arrow_right',
+    arrowsAreHidden && 'details-menu__arrow_hidden',
+    rightArrowDisabled && 'details-menu__arrow_disabled'
+  )
+
+  const scrollTabs = toRight => {
+    let scrollWidth
+
+    if (toRight) {
+      if (
+        tabsRef.current?.scrollWidth <
+        tabsWrapperRef.current?.offsetWidth * tabOffset + scrolledWidth
+      ) {
+        scrollWidth =
+          tabsRef.current?.scrollWidth - tabsWrapperRef.current?.offsetWidth
+
+        setRightArrowDisabled(true)
+      } else {
+        scrollWidth =
+          scrolledWidth +
+          tabsWrapperRef.current?.offsetWidth / menuOffsetHalfWidth
+      }
+    } else {
+      scrollWidth =
+        scrolledWidth -
+          tabsWrapperRef.current?.offsetWidth / menuOffsetHalfWidth <
+        0
+          ? 0
+          : scrolledWidth -
+            tabsWrapperRef.current?.offsetWidth / menuOffsetHalfWidth
```

Comment:
Note that `A < B ? B : A` is equivalent to `Math.max(A, B)`.

```suggestion
      scrollWidth = Math.max(
        0,
        scrolledWidth -
          tabsWrapperRef.current?.offsetWidth / menuOffsetHalfWidth
      )
```